fluxy/plots/mf_timeseries.py

```diff
@@ -266,8 +266,8 @@ def plot_timeseries(
                 # Define uncertainty band
                 flag_fill_between = False
                 if unc_var.split("_")[0] == "percentile":
-                    y1 = ds_plot[unc_var][0, :].values
-                    y2 = ds_plot[unc_var][1, :].values
+                    y1 = ds_plot[unc_var].isel(percentile=0).values
+                    y2 = ds_plot[unc_var].isel(percentile=0).values

```

Comment:
```suggestion
                    y2 = ds_plot[unc_var].isel(percentile=1).values
```
This should select the second value I think.

Reply:
Yes, sorry. Fixed now.